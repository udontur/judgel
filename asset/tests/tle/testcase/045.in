3228 191
97964 3445 31072 10142 51401 37796 516 35986 51156 87760 5563 3478 85493 11307 14880 67113 70480 10877 64480 99029 73944 12137 99346 95563 25682 10033 21075 25833 4931 91303 13914 41553 25364 917 56200 98068 17950 73617 78237 80663 63000 2098 45265 47733 72395 29174 38493 29026 79773 29268 96531 67903 8656 53063 57739 13614 43367 39277 72780 23366 41566 64057 3266 7497 15329 47378 71023 83892 87919 7121 94179 98672 26456 62556 52818 68092 35859 30489 22576 89629 60366 69509 85378 51859 94486 82033 32192 30309 64255 30552 95150 12663 15072 29022 23113 39198 14887 49928 30145 29029 93388 71298 62193 43195 89721 82891 97133 36828 27967 7306 72434 57560 93897 53279 89227 8902 27936 22617 19 10248 83379 88714 19680 46529 16103 12537 58494 77799 96319 50008 4990 28557 54005 78799 78808 640 88014 4981 24165 12861 70408 9882 75789 37095 80737 99305 56648 64245 57515 45957 46761 55599 65668 99391 62164 84620 17156 79518 4864 45126 10704 19074 81537 75937 23597 57058 20378 41667 33713 96734 54220 72754 8255 33680 46558 40170 31022 27878 96201 33050 22200 61386 47475 17803 72281 46597 55652 5751 64122 32439 73671 43633 82675 44893 9499 2962 58540 65054 84104 44832 96751 70177 4654 27161 28336 96689 96058 56581 2494 53531 42538 32575 59913 77941 66561 63203 15229 83144 79597 40887 93067 90293 69584 93411 59457 86694 30091 93370 30753 3078 10899 69521 99550 44597 77501 64359 115 26863 29446 78479 63260 51806 30562 52751 53950 62117 50495 5934 54104 72255 12588 11373 44602 46311 27610 63319 75025 40889 53358 7348 22479 7199 26725 82993 85177 20800 8471 5367 65432 70302 80393 97404 60830 62775 88184 41476 89272 38767 5731 98420 83747 11395 79244 29400 82238 59796 74988 1674 85206 34647 47393 76819 41401 2983 74790 1958 10375 8571 40210 79204 52657 99292 36355 83033 61573 60248 17545 44392 98555 75728 64302 95425 41574 46456 64183 84546 80189 94220 99125 39848 7850 26816 22495 22674 12478 30514 87223 67791 62744 7539 7472 94777 96901 41964 74500 11689 6590 20460 73580 11665 56981 91276 26111 12842 11440 17501 76898 75136 56399 72345 10030 64648 54872 54324 6951 67992 7396 21016 98999 50957 78541 18735 91216 21245 29885 70075 87625 60182 27356 18532 71479 851 18009 5665 80988 77150 91373 23029 41892 65441 93325 10377 79737 27716 65221 63991 59336 73109 77213 25159 73919 86653 25837 95081 50367 20206 31001 84234 76869 34778 52423 61179 28271 45386 19 64514 55797 71646 26228 84473 95065 82033 97226 77706 48631 70796 57445 9539 37481 41231 82067 9417 28943 99299 50912 77113 43 98236 75314 60380 52780 44721 12766 32345 75703 7331 64703 50995 99847 5279 71366 9385 82300 96897 19714 154 91822 58265 18284 49065 36633 15167 53402 4082 38797 17618 93604 11223 8879 94405 79219 17904 1601 69563 34214 55845 95750 53332 53835 5369 64643 62302 23487 71232 58716 49865 1912 69246 96664 38789 70751 76490 16000 74922 48645 72569 47805 43657 49062 55859 91645 48568 95092 52830 86609 24171 43525 596 41185 54775 62942 38949 76053 67567 67704 17009 6950 91194 39364 5956 93583 55353 81558 8742 37029 7120 1733 10362 89107 64255 26394 38952 68342 17677 20156 83001 99404 55327 8610 38737 24560 61048 50194 59635 84898 82270 26777 30998 68666 6428 51735 34898 3797 38383 91468 83428 92189 15872 89869 68047 80508 85101 70020 3656 29504 32743 24158 85920 39641 75165 50449 79210 43564 9083 9306 40095 71549 6007 16438 35731 74356 14174 13014 18386 64028 8307 76499 88099 75912 22805 36 378 97749 14288 58952 43166 28819 17919 39828 43629 22968 22928 8434 11468 24787 34546 83494 78047 62261 30976 95592 26921 15297 93591 77493 46892 71738 74612 13134 50255 26011 34138 39623 19742 59389 22991 62821 46328 46050 43839 19084 18879 62814 53226 15494 51232 93584 15273 15206 21106 60889 471 15014 77715 94093 83676 25400 70610 89313 94781 78108 11870 15793 65762 85871 19960 45660 56470 93971 42877 44476 79774 6685 16137 84356 3135 24052 44577 26754 78052 56526 86356 98321 22178 23240 31626 19595 84464 96278 91369 57158 17056 80001 94229 34783 87345 77484 2829 66631 95056 58481 7655 21015 34878 4047 91989 96173 96209 44689 30890 54314 84638 31313 83837 11562 5635 19804 38574 39810 61625 65555 21685 24611 55780 1064 84297 63611 5569 34768 65459 18908 6316 69982 54008 85475 97935 81957 60848 53993 60974 10991 63802 33219 11676 60748 73341 40666 72304 51470 82104 46358 8635 93144 19601 73246 92994 54309 74053 54930 95571 34141 7560 11864 47860 26509 5993 23034 90289 15517 90773 92305 65089 85001 80219 61918 21361 68160 64648 86835 58365 51265 42735 41896 61492 60383 8437 84747 60115 98452 18198 55370 16077 4946 69983 68846 2740 13106 9505 87403 48754 46329 70835 98158 71456 96178 60535 83271 17195 92309 23594 14095 73977 24876 74639 13939 13893 46137 46920 67677 41275 39958 77902 86033 76963 78299 72410 17584 12943 46362 80416 33093 26970 18007 31916 8442 61654 9802 17576 73142 39031 50833 88987 57313 96304 7574 82936 30804 14791 16048 26920 71330 4590 24901 73732 48574 67508 1994 73665 10433 15399 69377 98568 46223 1128 70124 28736 19537 90679 12754 42592 30719 54109 41678 66810 39998 9284 84725 26537 265 74827 57002 29501 35444 79873 32496 19318 72353 19145 58727 16202 22347 63743 97453 49761 58706 14695 85309 93072 97674 88145 91634 49563 76763 3745 70633 69573 32637 77196 67679 69634 71980 69465 48342 43282 21490 92306 14262 81989 2827 93674 17182 35903 4088 25722 93392 33695 23542 62076 98320 96726 15020 1374 16220 13542 5927 1015 75586 86582 81523 73175 77502 57222 44683 46776 45176 87464 65789 43217 39844 47015 57481 99217 59138 71902 79013 22683 21309 57452 81800 5634 41003 16581 332 24006 47523 29688 15989 13085 36989 74069 97368 92076 32507 96431 23238 66755 51802 54976 87095 1460 56873 54218 93357 81189 71269 16904 7621 38444 1271 14218 50259 71897 8722 32102 33099 62262 48451 34927 96197 10774 34751 40060 29815 6613 75791 88238 64131 79735 40640 70081 76719 2671 23735 58210 36035 19973 51010 4721 13861 35665 75622 70183 13215 19898 76944 76831 36312 96036 71415 24733 10996 20646 89295 60292 57361 45918 11551 61173 41256 20552 29326 19417 62169 35740 79001 33904 8850 6181 99254 15597 17845 49624 46816 84421 8297 84504 98016 36115 10783 91389 9844 94598 8932 79024 32014 62020 21274 58774 94677 69152 68347 9600 78101 98500 11722 82251 66580 34840 49565 36197 71087 98505 38917 76037 21797 50790 66891 20636 97507 36213 58475 79783 63937 37422 97807 84714 41729 79745 69830 23233 32294 65446 55175 61980 78534 5457 37993 16060 76233 2350 29882 39710 46901 57378 35550 99152 78275 74694 88237 3770 89089 47900 75653 54290 65628 99498 45200 96607 68201 30473 72539 58665 53288 53919 6036 44883 91995 69714 839 90555 63716 36279 33137 98415 78988 3812 27805 75015 43440 53237 24456 69365 66950 40538 15621 8691 85639 3952 62582 64227 46304 87859 62529 55525 42392 73059 38151 92250 45496 60091 68596 92037 69732 2160 55646 56679 83730 69933 76986 34779 114 62375 56487 32959 43824 97082 33449 83232 54397 86610 64810 96665 72341 68528 65480 86480 29820 68101 11606 75140 91979 83998 36746 43087 56176 57009 78893 95337 57956 2331 28061 78251 30019 78636 44084 19165 18053 3908 64296 46123 74303 86145 40459 25589 93716 31180 86365 18507 65872 49246 70784 68098 19823 45283 5884 60571 47999 96283 82936 71989 34384 53584 17326 54738 44163 17173 20027 46417 85496 76317 49513 72539 55448 17888 5268 1882 59770 97081 24478 21514 19691 59512 36311 13970 12786 5267 74646 98904 57044 84119 66660 64098 80281 42481 38870 56428 91159 94303 38725 64855 72639 96315 52783 40714 10545 75379 32761 10651 67 2337 77449 36714 60661 77602 54488 86348 24104 34272 76043 99956 33615 54877 15877 87118 99434 49697 50810 91953 29563 18118 38038 25265 27775 21784 66214 66405 66795 77185 97732 14885 87441 30360 45884 74456 99560 20671 22919 71973 36167 53447 89809 14449 59322 92000 32647 21725 26460 31069 41811 94652 6228 773 6076 52739 34788 7561 27981 61269 30509 77477 28741 25230 94675 5643 55959 99857 18132 10209 35985 53733 16863 8911 49880 61924 92338 76220 58518 57823 28658 82570 31538 14764 94400 38798 27836 89055 28238 64585 94713 75003 804 14234 76076 11764 35862 38832 64393 30431 55447 46403 21100 64259 27132 42229 88131 53472 99948 2584 34605 11449 89693 15559 47608 75348 90420 95259 66663 58749 25822 70424 49887 70627 25883 65011 51137 94958 63111 6906 8752 84555 55780 22804 6536 85138 15648 74865 62790 28456 74598 53582 82189 80857 9425 38875 25471 32688 32006 67893 68184 80591 45108 50709 53098 11177 20550 7800 51045 59056 92625 58071 52559 29521 71783 43770 35815 81706 1944 36480 97800 91019 5545 90200 95947 50763 42167 40753 27884 3920 44506 97164 27326 5672 81846 32518 24631 92664 17754 84606 23640 5858 97862 82386 35690 93603 82457 83763 70230 87219 16194 59399 15295 42277 91485 98335 32382 66896 2293 75731 57937 13753 6094 25243 6758 81141 61143 18724 70615 21150 85599 40769 68069 86912 34455 71748 38891 23256 32766 4542 25752 22830 48600 97489 46868 59690 69308 96101 6786 68173 24954 45219 74882 34829 83681 29213 10979 67370 90912 66277 35869 60067 70556 76613 30231 19527 3876 40317 54822 50633 5185 88625 91969 79070 96428 75708 77842 53606 42043 39252 53907 56859 56069 41386 38581 25540 99610 33107 53753 95400 89881 86326 76295 88287 34828 22945 51272 52851 28923 11480 58363 14545 33116 2764 57724 30154 45360 71471 24125 49231 89678 14734 48216 72382 77117 32004 16190 15195 59224 33976 86681 80613 6917 52303 18632 33066 13802 44910 29104 40069 34924 84908 89999 3913 27711 25937 92506 42005 99631 49414 79152 60121 83258 57392 54692 42454 15975 5065 98565 26605 36729 77021 57696 93036 24053 49334 45038 93765 47695 35793 26920 24825 56499 34315 52300 47950 38894 56761 41799 11148 6909 37676 2222 81922 71115 49012 61225 68010 49965 42747 98302 14804 86908 47672 41674 10217 30284 53159 491 90604 10709 95932 84842 9999 36634 60390 68357 60460 70075 7348 2006 1616 96672 99019 43264 11852 54514 88181 36870 42365 19215 99108 74135 57356 26834 24263 81013 20438 91742 48530 59389 1818 1821 27230 71386 69574 31513 22145 14693 62227 19841 75352 97444 41604 29277 86641 6916 290 79867 68080 15747 42612 32165 17443 9523 86047 59124 21500 88703 84602 77208 42947 57092 80605 91411 73091 77652 55731 56744 2368 57944 79145 60703 70118 42281 16385 39924 99779 48417 98031 33104 34236 87133 35589 87729 27066 9219 56354 96606 56842 65712 66205 98886 66016 83508 57860 41074 98772 82484 40878 49578 35814 28015 84798 65161 30495 79348 66356 66629 83116 53346 34957 52963 54682 46544 39903 58961 53513 31015 80301 51293 33604 59807 21289 94019 7876 28889 93160 80664 71392 40120 89395 77993 36718 80302 19850 87264 63468 95991 78513 52579 42702 29967 4466 48997 23079 31184 31020 94403 90035 50069 24243 50256 50006 4939 31895 52102 39710 38722 31065 33387 90878 61234 80693 18857 8321 82113 31444 90609 32057 95468 48498 93488 64773 45818 81121 84804 38612 55145 28988 42410 59050 11752 56508 85328 12302 72797 96006 69653 87064 48890 680 5759 86472 1699 20118 22730 68255 79697 70515 17162 305 67664 68429 69222 44866 50736 90467 36481 89515 19102 19048 80095 45780 97663 42822 49430 32803 14516 21674 69933 18534 78053 87033 97299 71159 3649 72634 70023 36795 95336 71225 17930 357 38546 82796 50321 84934 75010 93588 46831 23683 33680 69551 60319 70357 57323 91517 79521 50416 29034 99974 89220 54213 13853 19872 82903 86728 61181 14645 95092 59872 94324 20120 49985 25066 29239 67881 80025 2469 54419 34963 27005 87674 23708 82505 69377 93280 8353 29274 10507 59117 29708 78592 5692 31848 65182 18668 13010 47412 78521 78912 19392 80950 16530 24091 13625 50845 22417 32890 68130 2677 12801 93085 6637 4331 81783 50862 30026 85701 8536 36018 24768 44738 30198 20336 10042 67091 57364 42906 86067 8893 96402 46698 99077 80255 30421 82623 79231 86686 77071 83608 83433 29579 8443 72518 73478 17025 35707 42330 93207 41411 9023 44320 7015 86869 37707 2907 53183 50719 65180 34645 3705 5743 45726 27535 12577 70679 91870 59484 50419 94934 95297 25954 55939 92925 76297 97651 89360 13806 70204 83707 16188 55973 38919 57952 97288 97345 37548 87583 84430 95056 13696 93511 99617 63264 99784 71173 51597 17010 98418 48925 28050 8205 26080 41272 80091 81938 5544 58906 61568 56016 34264 31776 60638 43908 64841 64269 89783 21708 9618 25985 95120 98156 87172 80042 25557 67948 17905 78658 19546 6387 63732 62682 23955 99549 65974 31913 55438 75907 42371 93461 19874 58030 53408 26865 51660 57249 91611 56592 28469 1576 13594 88081 39747 39207 16465 21102 42696 66261 90432 61533 80637 71661 77970 31286 2950 83590 6969 42820 32963 16418 66830 21412 54211 98604 18284 46227 41444 67440 20434 70570 72367 78336 77871 97085 96060 22645 88304 73346 87328 12463 91377 130 6388 24697 68026 71740 18246 88192 24564 53963 21961 35380 80533 41055 73386 65146 19231 67638 72857 31277 66575 66411 5012 35679 50739 64222 35635 10278 60184 12821 73247 36456 26831 31103 45430 60598 45650 52648 36791 80113 91266 8955 87154 35764 34853 89290 35005 62468 67237 33600 49199 59364 61553 17217 90479 90658 97205 32370 94625 70640 44367 15354 1007 32269 84728 55346 45795 30155 98844 88892 28677 54573 76258 37788 49962 51928 98718 76521 9155 83664 30624 6211 92664 88671 64591 72813 54135 54754 77223 69846 98068 6262 24480 2305 23980 56479 30603 68334 96144 91640 77217 43789 15949 4308 20582 37455 10534 32160 14916 89776 29889 33346 70007 81003 66906 21873 69674 99141 46853 26385 1190 73097 8258 99254 48490 85899 54566 65635 23996 65160 30413 70082 30384 64824 67419 34487 47732 18722 82744 29119 68939 92210 44253 83046 1801 27647 77530 58414 3636 37790 23256 7053 32549 77150 58204 54908 3052 35714 80606 82246 14314 3028 84608 64927 83636 59963 17473 39912 31647 42103 5083 97846 67120 87201 33933 43882 75451 10964 75279 12322 42087 62199 59124 63117 88882 56126 95395 49023 5938 63953 94091 27225 92552 62502 52212 63695 884 25613 84794 68109 32541 77023 95157 98780 89247 16725 929 27120 24411 58007 24992 97037 97597 29678 32680 77208 11412 97698 63170 77822 3281 95958 30383 43852 60696 20012 98477 41370 3240 99963 10580 99055 95876 18194 77624 69863 5383 57445 80263 47264 74540 52522 23303 79858 24039 63213 70896 19066 35000 11462 58633 1090 51074 16758 49488 23488 36491 24571 26584 94774 72227 64304 80156 8750 32372 78164 78487 66052 40512 52791 68824 46926 26799 10673 19588 79947 14590 80613 27624 35708 31529 65531 59160 3451 5932 36258 29911 32498 81173 51782 59940 19249 94051 54316 92670 28492 50498 92116 76891 11779 43760 56333 83888 35571 16621 64225 27228 49474 92569 22683 31650 15552 78427 30138 66982 50978 15254 34532 85810 82740 65351 60366 66216 28734 25324 41558 7805 60672 42676 49419 6765 24176 98524 2435 65612 91141 40395 72176 43727 98584 18284 40159 11223 72909 6790 46142 41562 40207 19365 44025 31429 47710 55227 73625 24481 14662 86464 49906 59802 87195 14505 79072 27276 97334 65053 39212 30487 23139 7312 89674 97965 66304 476 66357 57374 33657 83491 58672 46890 72094 23893 76068 59814 95851 64086 7125 24863 66316 30754 13294 75361 40815 8043 72964 98277 55760 77647 31894 4637 43311 95514 35968 70616 95282 43915 49479 77911 12565 65265 39788 98317 68007 42808 8655 59079 83109 12961 7861 57928 88769 2400 41208 20977 90111 15612 58396 54861 50903 51048 2540 62111 59881 90343 18326 53226 60044 77051 73491 11437 26853 65996 69415 25219 44257 79892 48307 49239 79940 32226 32250 50600 46953 71053 79223 20683 49624 37397 93351 48366 77400 18666 54081 19380 11047 46202 25897 12273 51848 29892 37784 13959 93143 37676 34759 40137 89139 6359 58382 59416 99950 78465 64964 39307 8969 35619 31770 80037 440 7813 94311 29723 26630 74899 23092 17540 38389 17797 8655 91073 17778 51611 18550 5075 35531 18396 55208 94233 2820 40937 91659 38970 449 82919 39890 83576 14698 16673 15951 54120 36635 7831 12369 78493 23603 99805 55702 50872 71510 32463 48552 98044 45520 5503 83194 90294 98688 32909 88217 30828 12461 24759 70925 95112 57622 30360 7396 98104 58211 77210 21195 72977 2490 37488 997 22274 27108 25020 52697 22213 5283 26849 47452 93545 36642 57961 94532 83526 90811 72179 75432 99901 75642 68435 83565 5920 61449 52241 75363 28620 36177 37027 11043 90442 2256 8631 46967 32481 35975 96268 89858 55546 38495 74045 58190 71234 43762 934 66465 13274 51146 55296 27236 68100 77754 23409 12404 88403 82588 3493 84764 99917 44098 7292 78885 57179 89809 58912 29926 37523 73704 54788 92991 92412 97097 31762 90613 49925 83756 90033 4662 93915 78685 93434 19488 15302 43771 86226 47581 79050 91775 43277 1158 68050 75290 34082 7926 28040 56306 40734 2023 13926 18283 77870 88955 75659 56792 94325 38413 36278 49202 62462 84140 64910 52006 76846 41857 14363 30151 89664 62134 43345 71900 32462 21855 26543 62642 17058 77057 10953 83814 3335 9716 57638 32780 80627 56325 88498 74814 62694 45499 48181 65075 10768 74409 69589 91607 88860 12590 25852 49806 98322 80336 71653 3243 38844 30885 33895 46574 3508 98157 32375 83117 71335 30260 41752 48363 38544 52350 66277 64591 6985 53494 32619 11600 18559 42444 43830 93137 41361 24255 97260 30716 63967 98114 59972 73396 90739 5701 72380 28766 49957 72174 70706 54917 14716 12280 34329 16226 34209 70928 69392 69079 35967 28797 3240 44652 16396 12201 82541 66930 67400 96128 47692 98791 34471 66732 22186 93370 80160 79834 93778 61435 34518 94173 46248 67456 20531 45812 99981 38850 70004 54552 401 18818 10871 73720 44042 79509 78190 18428 20331 13057 16529 66700 65414 49902 60200 83821 92744 7815 70054 4220 70226 40080 76781 6813 30412 55425 78102 52594 22652 89643 54735 15506 99191 9340 54981 49407 45232 48822 8294 66807 74441 67109 20729 99191 37258 70314 63667 29704 99669 42745 24576 62166 28775 86109 9245 8778 11252 90354 31368 10253 72906 59015 62298 44066 16132 9079 32075 46784 77642 43254 55048 99922 76505 64923 50224 87645 19346 32215 77849 40686 44482 80164 27816 16139 58520 26183 86419 79243 20024 92618 14092 99759 48096 49781 4397 95559 80858 97834 95429 15002 3759 85003 4803 33674 19797 5610 68149 38093 15322 87813 63033 25100 97581 14895 17170 10386 13761 84039 99475 10495 1916 17215 81350 12577 12094 84022 31140 45665 7137 98026 14547 81338 25293 54269 31928 38225 68395 7007 36299 80309 30464 42053 80924 40282 73933 22389 48940 14043 4780 68421 89990 56937 82753 19431 82437 80319 39609 30244 77289 52448 6544 2843 49819 4540 19067 69840 40643 12763 56054 72125 29420 97609 51810 7240 91969 87516 64122 90308 58598 38388 96674 46850 29796 45526 22319 54539 93258 65504 42728 62745 26501 38976 65130 89905 55983 18106 69753 19274 84600 91806 81955 74592 11232 9014 3369 46520 42784 16503 95651 756 28309 3426 86755 25360 459 21200 61632 75158 43626 38885 4195 31673 22388 9147 5755 69216 76316 93578 67655 54712 83663 50781 84318 98701 10408 4249 73685 53195 50230 22604 28928 42720 3010 76192 67060 50534 72592 7945 1313 29594 87985 32872 20894 11152 74405 35694 16347 74115 68836 83084 31787 34046 90445 74244 53176 62311 75077 49357 50865 42263 16788 28229 70203 27626 62832 20413 69161 36365 43269 67183 72582 49987 2947 14361 95883 58141 61131 96920 69361 53401 17463 28274 90023 14253 64291 81826 36557 83460 68892 89007 23740 20476 34710 69512 12329 89541 7310 44590 84910 68518 49153 27464 10207 89871 84520 76160
64827 24418 8734 29858 47384 6966 5472 95765 37078 27935 25593 87558 7481 7119 83212 10876 11260 21399 53711 64406 46113 99183 33479 11221 72825 5590 46352 95250 94055 26358 84492 66548 30639 67501 32383 4422 77302 15387 40779 3460 97654 26945 35459 94002 7912 11618 71912 18104 58038 71267 92986 10623 81195 17507 41830 28501 35603 23469 86622 29310 28052 33043 78791 49615 38096 26596 15655 97742 89203 81849 56686 41778 51793 15575 74448 96005 60425 86213 81443 59488 70845 52280 97957 11844 68395 62241 21892 98067 190 57830 6349 89965 16983 61166 28527 3682 18644 46959 17795 7046 91829 4305 93483 70946 96357 63468 70667 81585 3958 71871 265 42238 31884 35113 54511 46337 52009 91525 4581 98092 89958 58073 23358 78488 45136 92286 63163 25023 49471 54429 53952 4137 56863 41618 42986 25052 58027 14857 36878 22896 48122 64537 44652 31611 17661 42705 43294 29337 44585 4271 83831 69917 11446 17166 4041 20199 69939 26899 95241 2001 59253 44154 76451 34987 98182 1747 39484 28339 6728 29576 71119 89526 7106 26953 72541 5900 75426 79862 51114 96125 13358 60809 51632 66107 13543 2714 51837 79357 21064 54649 52818
