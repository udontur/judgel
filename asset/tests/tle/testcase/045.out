13488621401262
